src/crimson/os/seastore/btree/fixed_kv_node.h

```diff
@@ -272,19 +272,25 @@ struct FixedKVNode : ChildableCachedExtent {
   get_child_ret_t<T> get_child(
     op_context_t<node_key_t> c,
     uint16_t pos,
-    node_key_t key)
+    node_key_t key,
+    extent_len_t e_off,
+    extent_len_t e_len)
```

Comment:
Do we have to touch `FixedKVNode::get_child()`? I think maybe we can keep the way of accessing extents' in-memory state, and only change the approach we access extents' data. How do you like it?